v2/CISCO-AAA-SERVER-MIB.my

```diff
@@ -1,1031 +1,1056 @@
+-- $Id$
```

Comment:
These `$something$` don't present any errors but seem superfluous?

Reply:
Resolving this, checked some other files and they are valid items although documentation is sparse on why we use them.

```
  ..\cisco-mibs\v2\CISCO-SDG-MDNS-MIB.my (4 hits)
	Line   1: -- $Id$
	Line   2: -- $Source$
	Line  11: -- $Log$
	Line  13: -- $Endlog$
```

```
 ..\cisco-mibs\v2\CISCO-RTTMON-MIB.my (4 hits)
	Line     1: -- $Id: CISCO-RTTMON-MIB.my,v 3.1.4.5 2022/08/30 17:34:50 sajg Exp $
	Line     2: -- $Source: /release/112/cvs/Xsys/MIBS/CISCO-RTTMON-MIB.my,v $
	Line    11: -- $Log: CISCO-RTTMON-MIB.my,v $
	Line    13: -- $Endlog$
```

```
  ..\cisco-mibs\v2\COMPAT-MIB.my (2 hits)
	Line  15:   -- $Revision: 1.11 $
	Line  16:   -- $Date: 2002/04/30 15:09:07 $
```